jmx-scraper/src/main/java/io/opentelemetry/contrib/jmxscraper/JmxScraper.java

```diff
@@ -106,31 +108,36 @@ private static void loadPropertiesFromPath(Properties props, String path)
   JmxScraper(JmxScraperConfig config) throws ConfigurationException {
     this.config = config;
 
-    try {
-      @SuppressWarnings("unused") // TODO: Temporary
-      JmxClient jmxClient = new JmxClient(config);
-    } catch (MalformedURLException e) {
-      throw new ConfigurationException("Malformed serviceUrl: ", e);
+    String serviceUrl = config.getServiceUrl();
+    if (serviceUrl == null) {
+      throw new ConfigurationException("missing service URL");
+    }
+    int interval = config.getIntervalMilliseconds();
+    if (interval < 0) {
+      throw new ConfigurationException("interval must be positive");
     }
+    this.client = JmxRemoteClient.createNew(serviceUrl);
+    this.service = JmxMetricInsight.createService(GlobalOpenTelemetry.get(), interval);
   }
 
-  @SuppressWarnings("FutureReturnValueIgnored") // TODO: Temporary
   private void start() {
-    exec.scheduleWithFixedDelay(
-        () -> {
-          logger.fine("JMX scraping triggered");
-          //            try {
-          //              runner.run();
-          //            } catch (Throwable e) {
-          //              logger.log(Level.SEVERE, "Error gathering JMX metrics", e);
-          //            }
-        },
-        0,
-        config.getIntervalMilliseconds(),
-        TimeUnit.MILLISECONDS);
+    try {
+      JMXConnector connector = client.connect();
+      connection = connector.getMBeanServerConnection();
+    } catch (IOException e) {
+      throw new IllegalStateException(e);
+    }
+    service.startRemote(getMetricConfig(config), () -> Collections.singletonList(connection));
     logger.info("JMX scraping started");
   }
 
+  @SuppressWarnings("unused")
+  private static MetricConfiguration getMetricConfig(JmxScraperConfig config) {
+    MetricConfiguration metricConfig = new MetricConfiguration();
+
```

Comment:
[for reviewer] this is where we need to create the new `MetricConfiguration` from the `JmxScraperConfig`.